python/ctsm/utils.py

```diff
@@ -31,5 +33,31 @@ def fill_template_file(path_to_template, path_to_final, substitutions):
         template_file_contents = template_file.read()
     template = string.Template(template_file_contents)
     final_file_contents = template.substitute(substitutions)
-    with open(path_to_final, 'w') as final_file:
+    with open(path_to_final, "w") as final_file:
         final_file.write(final_file_contents)
+
+
+def str2bool(v):
+    """
+    Function for converting different forms of
+    command line boolean strings to boolean value.
+
+    Args:
+        v (str): String bool input
+
+    Raises:
+        if the argument is not an acceptable boolean string
+        (such as yes or no ; true or false ; y or n ; t or f ; 0 or 1).
+        argparse.ArgumentTypeError: The string should be one of the mentioned values.
```

Comment:
- [ ] This documentation is wrong: it looks like it raises a ValueError, not an ArgumentTypeError

Reply:
Fixed.